packages/eas-cli/src/submissions/archive-source/ArchiveTypeSource.ts

```diff
@@ -101,11 +105,16 @@ async function handlePromptSourceAsync(
 
 type ArchiveInferredType = ArchiveType | null;
 
-function inferArchiveTypeFromLocation(location: string): ArchiveInferredType {
-  if (location.endsWith('.apk')) {
-    return ArchiveType.apk;
+function inferArchiveTypeFromLocation(
+  platform: SubmissionPlatform,
+  location: string
+): ArchiveInferredType {
+  if (platform === SubmissionPlatform.iOS) {
```

Comment:
Maybe sth like this would be cleaner:
```suggestion
  if (platform === SubmissionPlatform.iOS) {
  } else {
     if (...) {
     } else if () { 
     } 
  }
```
I know this is logically the same, but IMO this is easier to read.